recipe/meta.yaml

```diff
@@ -13,24 +13,31 @@ source:
 
 build:
   number: 0
-  skip: True  # [win or osx]
+  skip: true  # [win or osx or (linux64 and cuda_compiler_version != '9.2')]
```

Comment:
Ah, I guess we have such `True` issues in quite a few places.

Reply:
yeah I forgot: which one should we use and why?

Reply:
I think the yaml format accepts the lower-cased version, `true`. `True` came from the Python world.

Reply:
Actually, it does not care: https://yaml.org/type/bool.html.

Reply:
Ah OK. I'll then leave it as is. (I copied it from upstream.)